src/test/denialofservice_tests.cpp

```diff
@@ -393,6 +394,8 @@ static void ForceCoverageInPubKeyVerify()
 
 BOOST_AUTO_TEST_CASE(DoS_mapOrphans)
 {
+    SeedInsecureRand(ArithToUint256(arith_uint256(boost::unit_test::runtime_config::get<unsigned>( boost::unit_test::runtime_config::btrt_random_seed ))));
```

Comment:
I guess you could move this logic into a helper `SeedInsecureRandWithBoostRandomSeed`?